packages/connect-popup/src/index.tsx

```diff
@@ -418,13 +416,13 @@ const initCoreInPopup = async (
     setState({ core });
     log.debug('initiated core');
 
-    // init transport
-    /*
-    log.debug('initiating transport with settings: ', payload.settings);
-    reactEventBus.dispatch({ type: 'loading', message: 'initiating transport' });
-    await initTransport(payload.settings);
-    if (disposed) return;
-    */
+    // init transport - deprecated, here for backward compatibility
+    if (initTransport) {
+        log.debug('initiating transport with settings: ', payload.settings);
+        reactEventBus.dispatch({ type: 'loading', message: 'initiating transport' });
+        await initTransport(payload.settings);
```

Comment:
is there really some backwards compatibility issue? this all happens on trezor.io domain

Reply:
Because of custom `connectSrc`?

Reply:
ok, `connectSrc` or older cache could be it probably